install_cloud.json

```diff
@@ -0,0 +1,43 @@
+{
+  "builders": [
+    {
+      "type": "openstack",
+      "endpoint_type": "publicURL",
+      "ssh_username": "centos",
+      "ssh_pty": "true",
+      "image_name": "packer-demo-{{timestamp}}",
+      "source_image": "0681950f-6786-4286-896d-ff79e0047e0b",
+      "flavor": "8e00bc89-f2b0-40d3-96d4-05bd67f6e6bf",
+      "use_floating_ip": "true",
+      "floating_ip": "37.153.138.59"
+    }
+  ],
+   "provisioners":[
+     {
+      	"type": "shell",
+      	"execute_command": "{{.Vars}} /bin/sh -x '{{.Path}}'",
+        "script":"scripts/cloud/install-docker.sh"
```

Comment:
I suggest to package all installation steps (docker, docker-bootstrap, nettools) in one file (prepare-docker.sh). This way the installation can be executed with:

 curl -fsSL https://github.com/..../prepare-docker.sh | sh 
curl -fsSL https://github.com/..../master.sh | sh 


on any CentOS installation. Just want to keep all nicely decoupled.

Also, package the minimizing scripts in one file minimize-cloud-image.sh